support/docker/production/Dockerfile.arm

```diff
@@ -0,0 +1,65 @@
+FROM arm64v8/debian:stretch-slim
+
+RUN set -ex; \
+    if ! command -v gpg > /dev/null; then \
+      apt-get update; \
+      apt-get install -y --no-install-recommends \
+        gnupg \
+        dirmngr \
+      ; \
+      rm -rf /var/lib/apt/lists/*; \
+fi
+
+# install nodejs and npm
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update \
+&& apt-get install -y apt-utils nano htop curl git build-essential gcc g++ make \
+&& curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+&& apt-get install -yq nodejs \
+&& rm -rf /var/lib/apt/lists/ \
+&& npm install npm@latest -g && npm install yarn@latest -g
+
+# Install dependencies
+RUN apt-get update \
```

Comment:
apt-get start to be more and more depreacated, you should consider switching to aptitude (will probbably change nothing since most of apt-get are now symbolink to aptitude)
For your script that means replacing each apt-get to apt

Reply:
Should I update the Dockerfile.strech in the same time?

Reply:
I havn't tested it for your docker but aptitude is in debian way beffore stretch, so a replacement of `apt-get` to `apt` should be complete.

Reply:
Done in all dockerfiles